android/src/test/java/com/instabug/reactlibrary/RNInstabugRepliesModuleTest.java

```diff
@@ -170,6 +173,25 @@ public Boolean answer(InvocationOnMock invocation) throws Throwable {
         Replies.setPushNotificationRegistrationToken("123");
     }
 
+    @Test
+    public void givenBoolean$showNotification_whenQuery_thenShouldCallNativeApi() {
+        // given
+        PowerMockito.mockStatic(Replies.class);
+        // when
+        ReadableMap readableMap = null;
+        rnModule.showNotification(readableMap);
+        // then
+        Map<String,String> map = new HashMap<String,String>();
+        if(Replies.isInstabugNotification(map)){
+            PowerMockito.verifyStatic(VerificationModeFactory.times(1));
+            Replies.showNotification(map);
```

Comment:
Adding if conditions to tests will probably lead to unpredicted results. If you want to test two scenarios, please create a test for each one, each test should have the expected input data set beforehand to avoid if conditions.